README.md

```diff
@@ -16,6 +16,7 @@ Before you begin you will need:
 Clone and install dependencies:
 
 ```bash
+npm install -g concurrently
```

Comment:
This should be in package.json as a Dev dependency rather than something you have to manually install.